### DISCUSSION TITLE
database/mysql/mysql-questions-01

### DISCUSSION THREAD

**Discussion by giscus** [Snailclimb/JavaGuide#1662] [Announcements]
# database/mysql/mysql-questions-01

Java学习&&面试指南

https://javaguide.cn/database/mysql/mysql-questions-01.html

**Comment by TandL521** [comment]
为啥有些图片显示不了啊？

**Reply by Snailclimb** [reply]
刷新下试试，我这边显示没问题。

**Comment by fengxiaop** [comment]
```。 例如：事务 1 读取某表中的数据 A=20，事务 2 也读取 A=20，事务 1 修改 A=A-1，事务 2 也修改 A=A-1，最终结果 A=19，事务 1 的修改被丢失。``` 因为是最终结果应该是18  但是因为事务2把事务1的所覆盖了 所以才导致事务1的修改丢失么？

**Comment by TandL521** [comment]
我这边是最近都出现了刷新没有解决问题。



发自我的iPhone


------------------ Original ------------------
From: Guide ***@***.***&gt;
Date: Mon,Apr 4,2022 8:39 PM
To: Snailclimb/JavaGuide ***@***.***&gt;
Cc: TandL521 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [Snailclimb/JavaGuide] database/mysql/mysql-questions-01 (Discussion #1662)





 
刷新下试试，我这边显示没问题。

**Reply by Snailclimb** [reply]
1、确认是否是 HTTPS 访问
2、如果还是不可以，请访问：https://gitee.com/SnailClimb/JavaGuide/blob/main/docs/database/mysql/mysql-questions-01.md 。

你这个问题我目前是不清楚是什么情况导致的，我没遇到过，其他读者也没有这种问题，没办法复现。建议你看看浏览器控制台，搞清楚到底是什么问题，截图发我看看，这样我也好去排查一下。
